CMakeLists.txt

```diff
@@ -321,8 +323,18 @@ if(${HAVE_BABELTRACE})
   set(WITH_BABELTRACE ${BABELTRACE_FOUND})
   set(HAVE_BABELTRACE_BABELTRACE_H ${BABELTRACE_FOUND})
   set(HAVE_BABELTRACE_CTF_EVENTS_H ${BABELTRACE_FOUND})
+  set(HAVE_BABELTRACE_CTF_ITERATOR_H ${BABELTRACE_FOUND})
 endif(${HAVE_BABELTRACE})
 
+option(DEBUG_GATHER "C_Gather debugging is enabled" OFF)
+option(HAVE_KINETIC "Kinetic Client is enables" OFF)
+option(HAVE_LIBZFS "LibZFS is enabled" OFF)
+option(ENABLE_COVERAGE "Coverage is enabled" OFF)
+option(PG_DEBUG_REFS "PG Ref debugging is enabled" OFF)
+option(HAVE_ARMV8_CRC "Supports ARMv8 CRC instructions" OFF)
```

Comment:
@dmick I set HAVE_ARMV8_CRC as an option and set it to off just to throw something over the wall. We have builds going on ARMv8 and should something in CMake for platform detection right?